### PR TITLE
Add initial support for php70 #7

### PR DESCRIPTION
**Pull request by mirfilip**
Hi,
just a quick take on mentioned task. Please review it, it has been ages since I have done any scripting in bash.

Two things to mention:
+ I thought about naming convention for `php7x`. Even though it's still beta, I went on with current conversion and didn't do anything like `70b` or similar.
+ As for php7x apache module name and key for `httpd.conf` got changed, I needed some conditional logic.

Also, I noticed there is something broken with substitution of `LoadModule php5_module ...` in `httpd.conf` in HEAD. It's not related to my changes. I didn't change any logic except naming, but I will revisit the matter later.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Your Apache config must be setup using the same paths as below. Replace homebrew
 Version
 ----
 
-1.4
+1.5
```
Comment (mirfilip) [inline README.md]: I though you'd like to release the new version if that PR comes through.

File: README.md [hunk 2]
```diff
@@ -44,5 +44,5 @@ License
 
 MIT
 
-[Homebrew]:http://http://brew.sh/
-[@p_cook]:http://twitter.com/p_cook
+[Homebrew]: http://brew.sh/
```
Comment (mirfilip) [inline README.md]: links notation was broken

**Comment by akrabat** [comment]
Any news on if this is likely to be merged?

**Comment by philcook** [comment]
@akrabat apologise I have been away on holiday and not been able to review this. I will take a look this week.

**Comment by akrabat** [comment]
Thanks @philcook :)

File: phpswitch.sh [hunk 3]
```diff
@@ -68,9 +74,14 @@ then
 			echo "Switching your apache conf"
 			for j in ${php_installed_array[@]}
 			do
-				sudo sed -i.bak "s/^LoadModule[ \t]php5_module[ \t]$php_opt_path$j$apache_php_lib_path/\#LoadModule php5_module $php_opt_path$j$apache_php_lib_path/g" $apache_conf_path
```
Comment (philcook) [inline phpswitch.sh]: @mirfilip this part is required to comment out all existing PHP modules in apache conf not just the one being selected.

**Comment by philcook** [comment]
@mirfilip checked out your code and unfortunately it doesnt quite work due to the fact it doesnt comment out any other PHP module lines in the apache code.

However I have done some work myself and put it in a [php7 branch](https://github.com/philcook/brew-php-switcher/tree/feature/php7).

This version has a few advantages as well that I have been meaning to put in. Mainly that if the PHP module apache conf line its already in the apache conf it will now add it for you.

I will merge mine into master shortly but in the meantime if you want to checkout this branch your more than welcome to.

**Pull request closed without merging**